diff --git a/configure/os/CONFIG_SITE.Common.linux-x86_64 b/configure/os/CONFIG_SITE.Common.linux-x86_64
index b98385f16..d9b648125 100644
--- a/configure/os/CONFIG_SITE.Common.linux-x86_64
+++ b/configure/os/CONFIG_SITE.Common.linux-x86_64
@@ -41,3 +41,25 @@ GNU_TUNE_CFLAGS = -mtune=generic
 
 # Developers using a suitable compiler may enable its address sanitizer:
 #ENABLE_ASAN = YES
+
+
+CFLAGS+= -isystem /twin/miniconda3/include
+CXXFLAGS+=$(CFLAGS)
+CCCFLAGS+=$(CXXFLAGS)
+
+CFLAGS+=-fno-strict-overflow \
+          -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O3 -ffunction-sections \
+          -pipe  -isystem /twin/miniconda3/include \
+          -fdebug-prefix-map=/home/conda/feedstock_root/build_artifacts/python-split_1708115583107/work=/usr/local/src/conda/python-3.12.2 \
+          -fdebug-prefix-map=/twin/miniconda3=/usr/local/src/conda-prefix \
+          -fuse-linker-plugin -ffat-lto-objects -flto-partition=none -flto -O3 -Wall
+CXXFLAGS+=$(CFLAGS)
+CCCFLAGS+=$(CXXFLAGS)
+
+
+LDFLAGS+=-L/twin/miniconda3/lib/python3.12/config-3.12-x86_64-linux-gnu -L/twin/miniconda3/lib
+MINICONDA_PATH=/twin/miniconda3/bin/
+CC=$(MINICONDA_PATH)/gcc
+CXX=$(MINICONDA_PATH)/g++
+CCC=$(CXX)
+LD=$(CXX)
